cmd/validate/image_test.go

```diff
@@ -794,7 +810,8 @@ configuration:
 			"registry/policy:latest"
 		],
 		"ruleData": {
-			"key":"value"
+			"key": "---\nkind: ReleasePlanAdmission\nspec:\n  application: [some-app]\n  data:\n    mapping:\n\t  components:\n\t    - name: some-name\n\t\t  repository: quay.io/some-namespace/msd\n",
```

Comment:
Did you consider unmarshalling it? My thinking is it's more flexible not to, but since you're only doing this for a .json or .yaml maybe it makes sense.

Related question, I wonder if a user would ever want to slurp in some text from a .txt file..?

Reply:
hm, please correct me if I am missing something but it does look like it gets unmarshalled later on here: https://github.com/enterprise-contract/ec-cli/pull/1706/files#diff-7f11215ba891ff8199fa09a412849a5414e416050d163c813f6499009bbe0b96R271 (Line 271)

I did not consider .txt file in the scope of this change as we don't have a use case for it. Do you think we should add it now?